src/angularComponent/ngcOmniboxSelectionsDirective.js

```diff
@@ -5,12 +5,17 @@ export default function ngcOmniboxSelectionsDirective() {
       omnibox: '^^ngcOmnibox'
     },
     scope: true,
+    bindToController: {
+      selection: '<'
+    },
+    controller: () => {},
     compile(tElement) {
       const element = tElement[0];
       const tokens = element.firstElementChild;
 
       if (tokens) {
         tokens.setAttribute('ng-repeat', 'selection in omnibox.ngModel');
+        tokens.setAttribute('selection', 'selection');
```

Comment:
No, it's getting `selection` from the `ng-repeat` on the line above it. I'm storing that selection as a binding on the controller so that it can be retrieved on the click event to pass to the remove function.